doc/01-installation.md

```diff
@@ -0,0 +1,140 @@
+# Shopware OpenTelemetry configuration
```

Comment:
https://developer.shopware.com/docs/guides/hosting/configurations/observability/opentelemetry.html#opentelemetry

please change it there